server/api/bots.js

```diff
@@ -18,12 +19,13 @@ router.get("/:id", async (req, res) => {
 });
 //posts a bot to the db
 router.post("/", async (req, res) => {
-    const { id, name, prefix, description, owners, website, helpCommand, supportServer, library } = req.body;
```

Comment:
Why'd you remove name?

Reply:
```js
 const botApiData = await getBotData(id);
    const { tag, avatarUrl } = botApiData;
    if (bot) return res.status(400).send({ message: "This bot already exists!", error: "Bad Request." });
    const newBot = new Bots({ id, name, prefix, description, owners, website, helpCommand, supportServer, library });
    const newBot = new Bots({ id, tag, avatarUrl, prefix, description, owners, website, helpCommand, supportServer, library });

```
I am getting the full tag from here using the discord api so we dont need a name